src/main/java/core/basesyntax/service/ShopServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service;
+
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class ShopServiceImpl implements ShopService {
+    private OperationStrategy operationStrategy;
+    private Map<String, Integer> repository;
+
+    public ShopServiceImpl(OperationStrategy operationStrategy) {
+        repository = new HashMap<>();
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
```

Comment:
The 'process' method should not begin with an empty line. Please remove the redundant empty line at the start of the method definition.